docs/source/api/tutorials/In Depth 3 - Drawing DiHypergraphs.ipynb

```diff
@@ -13,7 +13,7 @@
    "id": "ea8aad6a",
```

Comment:
<p>is there a way to have the old "iterations=0" when wanted?</p><p>If not, maybe open an issue to re-implent it.</p><p>I like that in small cases, that one had the edge_markers at the barycenters. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xgi-org/xgi/pull/492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
I added a function called `edge_positions_from_barycenters` that takes in a dictionary of nodal positions and then returns a dictionary of edge positions computed from the barycenters.

Reply:
This is good because it is no longer tied to the specific position calculation (before, it was spring layout). Now you can use any of the node position methods (circular, spiral, Kamada-Kawai, etc.) to get edge positions from the barycenters.